src/test/java/com/unzer/payment/business/paymenttypes/PaylaterInvoiceTest.java

```diff
@@ -0,0 +1,484 @@
+                                                .setAmountDiscountPerUnitGross(BigDecimal.ZERO)
+                                                .setAmountPerUnitGross(new BigDecimal("100.1"))
+                                                .setTitle("Apple iPhone")
+                                ),
+                        getMaximumCustomerSameAddress(generateUuid()),
+                        (Authorization) new Authorization()
+                                .setAmount(BigDecimal.valueOf(500.5))
+                                .setCurrency(Currency.getInstance("EUR"))
+                                .setReturnUrl(unsafeUrl("https://unzer.com"))
+                                .setAdditionalTransactionData(
+                                        new AdditionalTransactionData()
+                                                .setRiskData(
+                                                        new RiskData()
+                                                                .setConfirmedAmount(2569.0)
+                                                                .setConfirmedOrders(14)
+                                                                .setRegistrationLevel(RiskData.RegistrationLevel.REGISTERED)
+                                                                .setCustomerGroup(RiskData.CustomerGroup.GOOD)
+                                                                .setRegistrationDate(new Date())
+                                                )
+                                )
+                ),
+                new TestCase(
+                        "no risk data",
+                        new PaylaterInvoice(),
+                        new Basket()
+                                .setTotalValueGross(new BigDecimal("500.5"))
+                                .setCurrencyCode(Currency.getInstance("EUR"))
+                                .setOrderId(generateUuid())
+                                .addBasketItem(
+                                        new BasketItem()
+                                                .setBasketItemReferenceId("Artikelnummer4711")
+                                                .setQuantity(5)
+                                                .setVat(BigDecimal.ZERO)
+                                                .setAmountDiscountPerUnitGross(BigDecimal.ZERO)
+                                                .setAmountPerUnitGross(new BigDecimal("100.1"))
+                                                .setTitle("Apple iPhone")
+                                ),
+                        getMaximumCustomerSameAddress(generateUuid()),
+                        (Authorization) new Authorization()
+                                .setAmount(BigDecimal.valueOf(500.5))
+                                .setCurrency(Currency.getInstance("EUR"))
+                                .setReturnUrl(unsafeUrl("https://unzer.com"))
```

Comment:
suggestion: Do you think the createion of the default resources like basket and authorization can be moved to a method and reused here, since it looks like they are identical in most of the testcases.(similar like it is done with customer)